scripts/generate_swagger.sh

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env bash
+
+set -euo pipefail
+
 PATH_TO_PROJECT_ROOT=$(git rev-parse --show-toplevel)
 SWAGGER_DIR="${PATH_TO_PROJECT_ROOT}/pkg/swagger"
 WEBUI_PATH="${PATH_TO_PROJECT_ROOT}/webui/lib/api/schema"
```

Comment:
In the case that swagger isn't installed can we add
```
# Check if 'swag' is installed and install it if not
if ! command -v swag &> /dev/null; then
    echo "'swag' not found. Installing 'swag'..."
    go install github.com/swaggo/swag/cmd/swag@latest
    if [[ $? -ne 0 ]]; then
        echo "Failed to install 'swag'. Exiting."
        exit 1
    fi
    echo "'swag' installed successfully."
fi
```

Reply:
Yeah good suggestion. Sorry I merged before setting this comment